planetiler-core/src/test/java/com/onthegomap/planetiler/PlanetilerTests.java

```diff
@@ -1710,6 +1711,55 @@ void testIssue509LenaDelta() throws Exception {
     ), counts);
   }
 
+  @Test
+  void testIssue546Terschelling() throws Exception {
+    Geometry geometry = new WKBReader()
+      .read(
+        new InputStreamInStream(Files.newInputStream(TestUtils.pathToResource("issue_546_terschelling.wkb"))));
+    geometry = GeoUtils.worldToLatLonCoords(geometry);
+    Files.writeString(Path.of("test.wkt"), new WKTWriter2().write(geometry));
```

Comment:
Was it intentional to check in `test.wkt`?

Reply:
good catch! that was for testing... fixed